### ISSUE TITLE
Bug in challenge: Regular Expressions: Check for All or None

### ISSUE THREAD

**Issue by enigma0160** [freeCodeCamp/freeCodeCamp#38641]
**Describe your problem and how to reproduce it:**
This passes:
```js
let favRegex = /favou?rite/i;
```
But this doesn't:
```js
let favRegex = /favou?rite/gi;
```
with the error message: 
```
// running tests
Your regex should match "favorite"
Your regex should match "favourite"
// tests completed
```
**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/regular-expressions/check-for-all-or-none

**Tell us about your browser and operating system:**
* Browser Name: Google Chrome
* Browser Version: 81.0.4044.113 (Official Build) (64-bit)
* Operating System: Fedora 31

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

**Comment by moT01** [comment]
I believe the method used to solve a similar problem in [this PR](https://github.com/freeCodeCamp/freeCodeCamp/pull/37941) would work here.

**Comment by Moonsolol** [comment]
Hi, could I try working on this issue? Thanks.

**Comment by muskeinsingh** [comment]
Is this still open? Can I pick this up?

**Comment by Sky020** [comment]
@muskeinsingh , Sorry, this is not longer open. It was closed with PR #38659 .
